### DISCUSSION TITLE
user-guide/faqs/get-more-style-cannot-open

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#156] [文档评论区]
# user-guide/faqs/get-more-style-cannot-open

“获取更多样式”打不开怎么办 近期许多同学在添加引文格式的时候，却发现 Zotero 内部 获取更多样式 点了但是却一直没有反应，可以通过尝试以下三种方法解决问题： 点击“获取更多样式”没有反应点击“获取更多样式”没有反应 禁用插件 这种问题可能是由于安装的插件发生冲突所导致，点击 工具-附件组件后禁用第三方插件后，重启 Zotero 该问题即可解决。...

https://zotero-chinese.com/user-guide/faqs/%E2%80%9C%E8%8E%B7%E5%8F%96%E6%9B%B4%E5%A4%9A%E6%A0%B7%E5%BC%8F%E2%80%9D%E6%89%93%E4%B8%8D%E5%BC%80%E6%80%8E%E4%B9%88%E5%8A%9E.html

**Comment by yuanyuan021006** [comment]
感谢！！！
